CSCAutomateUnitTest/CloudSkillApiTests.cs

```diff
@@ -29,10 +29,10 @@ public async Task AddLearnerAsyncTest()
             Configuration config = await TestHelper.GetConfigurationAsync();
             CloudSkillApi cscApi = new CloudSkillApi(config);
 
-            string learnerRequestJson = ContestFactory.CreateLearnerRequestJson();
-            string learnerReponse = await cscApi.AddLearnerAsync(learnerRequestJson);
+            //string learnerRequestJson = ContestFactory.CreateLearnerRequestJson();
+            Learner response = await cscApi.AddLearnerAsync("b95e57ea-61d1-4844-935e-7477e05d8f6a", "SriniAmbati-6212");

```

Comment:
Are we sure about hardcoding a challengeId here?
It's possible the challenge expires, especially when we change envs.
This API is also not Idempotent.